index.js

```diff
@@ -56,7 +57,8 @@ exports.build = async ({
       ...applicationFiles
     },
     handler: 'launcher.launcher',
-    runtime: runtime
+    runtime,
+    ...memory ? { memory } : {}
```

Comment:
Are we defaulting it? I need to check.

Reply:
> Hey, this is pretty cool! I have been tinkering how we could make using vercel-sapper easier and this seems like a good start. I'm not sure though if this is easier than manually addng the files and code. I imagine the perfect experience (apart from built-in support) would be to run `npx vercel-sapper` on an existing sapper project. What do you think?

The reason I don't like this approach is that it requires manual maintenance of dependencies and such as the template is updated. It's the same reason the svelte TS template is a patch rather than yet another unmaintained fork.

Reply:
What manual maintenance would be required? I think it would run the same install.js but it saves the user from having to clone this repo, running npm install, cd into the examples directory.

Reply:
Who would be responsible for manually merging the original repo into this one every time it is updated?

Reply:
No manual merging required, check it out! https://github.com/thgh/vercel-sapper/pull/47